### ISSUE TITLE
Plugin should suppress PropertyNotSetInConstructor on Response

### ISSUE THREAD

**Issue by laurentmuller** [psalm/psalm-plugin-symfony#360]
I have derived the `Response` class like as follow:

```php
public class PdfResponse extends Symfony\Component\HttpFoundation\Response
{
    // ...
}
```

 `PropertyNotSetInConstructor` is raised for the following properties:

- `$content`
- `$headers`
- `$statusCode`
- `$statusText`
- `$version`

If You check the class constructor of the `Response`:

```php
public function __construct(?string $content = '', int $status = 200, array $headers = [])
{
    $this->headers = new ResponseHeaderBag($headers);
    $this->setContent($content);
    $this->setStatusCode($status);
    $this->setProtocolVersion('1.0');
}
```

You can see that the properties are set.

This is bit the same as https://github.com/psalm/psalm-plugin-symfony/issues/359
